tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/transform/OperationContextCppCodeGenerator.java

```diff
@@ -0,0 +1,103 @@
+package com.amazonaws.util.awsclientgenerator.transform;
+
+import lombok.Data;
+
+import java.text.MessageFormat;
+import java.util.Stack;
+import java.util.Optional;
+import software.amazon.smithy.utils.Pair;
+
+@Data
+public final class OperationContextCppCodeGenerator{
+
+    private final StringBuilder cppCode;
+    //tracking reference variable, bool = true means start of a scope
+    private final Stack<Pair<String,Boolean> > varName;
+
+    private Integer codeIndetation;
```

Comment:
use primative `int` instead of `Integer`

Reply:
ack